src/diem/offchain/client.py

```diff
@@ -135,7 +135,21 @@ def send_request(
         return cmd_resp
 
     def process_inbound_request(self, request_sender_address: str, request_bytes: bytes) -> Command:
```

Comment:
can use https://pypi.org/project/deprecation/ for deprecation; `@deprecated` is nice

Reply:
the only thing preventing using these tiny handy packages are: we need minimum the dependencies if possible. 

1. we need verify security for all dependencies.
2. we need make sure fbcode has the lib too......